tutorials/setup.md

```diff
@@ -0,0 +1,231 @@
+# Get started with CRI-O
+
+This guide will walk you through the installation of [CRI-O](https://github.com/cri-o/cri-o), an Open Container Initiative-based implementation of [Kubernetes Container Runtime Interface](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/container-runtime-interface-v1.md).
+
+It is assumed you are running a linux machine.
+
+### Runtime dependencies
```

Comment:
It's more a matter of taste, but I would not skip the 2nd header here (`##`), so:
```suggestion
## Runtime dependencies
```